runtime/doc/provider.txt

```diff
@@ -236,6 +236,22 @@ The "copy" function stores a list of lines and the register type. The "paste"
 function returns the clipboard as a `[lines, regtype]` list, where `lines` is
 a list of lines and `regtype` is a register type conforming to |setreg()|.
 
+							      *clipboard-wsl*
+For Windows WSL, try this g:clipboard definition:
+>
+    let g:clipboard = {
+                \   'name': 'WslClipboard',
```

Comment:
This is from https://www.reddit.com/r/neovim/comments/g94zrl/solution_neovim_clipboard_with_wsl/ . Can someone using WSL confirm that this works? Or is there a better solution these days?

Reply:
I tried it out, it works but there is a non-insignificant delay for me personally when pasting. Using win32yank removes most of the delay, so maybe it could be worth mentioning?

```
let g:clipboard = {
            \   'name': 'WslClipboard',
            \   'copy': {
            \      '+': 'win32yank.exe -i --crlf',
            \      '*': 'win32yank.exe -i --crlf',
            \    },
            \   'paste': {
            \      '+': 'win32yank.exe -o --lf',
            \      '*': 'win32yank.exe -o --lf',
            \   },
            \   'cache_enabled': 0,
            \ }
```

 I will leave it up to you what to do with this information :)

Reply:
Derp, ignore what I said. win32yank is apparently already automatically used if it's installed so the above code snippet doesn't seem to be needed.